corehq/apps/export/views.py

```diff
@@ -1251,7 +1345,7 @@ def fmt_export_data(self, export):
             'addedToBulk': False,
             'exportType': export.type,
             'emailedExports': emailed_exports,
-            'editUrl': reverse(EditCustomCaseExportView.urlname,
+            'editUrl': reverse(edit_view.urlname,
```

Comment:
absolutely. but this is the next piece of functionality that i'm working on. essentially it takes the id and if it finds the old schema, it takes it and converts it to the new. probably easier said than done